site/components/pages/errors/styles.ts

```diff
@@ -0,0 +1,20 @@
+import { css } from "@emotion/css";
+import breakpoints from "@klimadao/lib/theme/breakpoints";
+
+export const fullheight = css`
+  min-height: calc(
+    100vh - (var(--footer-height) / 2 + var(--header-height) / 2)
```

Comment:
The fixed footer height creates a scroll bar as the footer isn't always exactly 15rem :( It's also somewhat fragile, if we add more content/icons to the footer, or change logo size or something.

Grid can do this dynamically, via `GridContainer` CSS:

```css
/* mobile */
min-height: 100vh;
grid-template-rows: 1fr auto;
  
/* desktop */
grid-template-rows: auto 1fr auto;
```

Problem: GridContainer is rendered in `_app`, so this would break some pages that rely on the implicit row sizes.
So maybe you can try wrapping just this page in a div and applying the grid?

```css
.404PageWrapper {
  display: grid;
  grid-column: full;
  grid-template-columns: inherit;
  min-height: 100vh;
  grid-template-rows: 1fr auto;
}
```

Reply:
Thanks for the grid. I start liking it more and more. Is changed.